nx.json

```diff
@@ -1,88 +1,162 @@
 {
   "namedInputs": {
-    "default": ["{projectRoot}/**/*", "sharedGlobals"],
-    "sharedGlobals": ["{workspaceRoot}/.github/workflows/ci=github.yml"],
+    "default": [
```

Comment:
I've fine-tuned the caching a bit.

these are simply "groups" of files. So these are files we say should, or should not be considered as inputs to a given command. This is reflective in caching.

For example:

`"!{projectRoot}/LICENSE",` - Don't consider this file as an input. i.e. we shouldn't care if this file changes as it relates to `test`, `build`, `e2e` etc. so therefore this file can change and we should not bust the cache.

`"{projectRoot}/**/*",` - Consider all files as input